staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -196,6 +196,16 @@ var (
 		[]string{"verb", "resource", "subresource"},
 	)
 
+	requestFilterDuration = compbasemetrics.NewHistogramVec(
+		&compbasemetrics.HistogramOpts{
+			Name:           "apiserver_request_filter_duration_seconds",
+			Help:           "Request filter latency distribution in seconds, for each filter type",
+			Buckets:        []float64{0.0, 0.0005, 0.001, 0.005, 0.01, 0.05, 0.1, 0.25, 0.50, 1.0, 5.0},
```

Comment:
Roughly exponential progression: good.
Note: the square root of 10 is about 3, not 5 or 2.

Reply:
How about
```suggestion
			Buckets:        []float64{0.0, 0.0003, 0.001, 0.003, 0.01, 0.03, 0.1, 0.3, 1.0, 5.0},
```

Reply:
sounds good, I made the change.